src/components/index.js

```diff
@@ -1,3 +1,5 @@
 import { ErrorBoundary } from './ErrorBoundary';
+import Header from './Header';
 
 export { ErrorBoundary };
+export default Header;
```

Comment:
why are you exporting default header from components? There can only be one default export. Header should be a named export.

Reply:
fixed